### ISSUE TITLE
`getBackgroundStack` gets confused between height and width

### ISSUE THREAD

**Issue by alice** [dequelabs/axe-core#292]
https://github.com/dequelabs/axe-core/blob/b45ae65dd2a8a9d7413a622b8f0f879451dac9e7/lib/commons/color/get-background-color.js#L62

```js
	if (rect.top > window.innerWidth) {  // <-- should be window.innerHeight!
		return;
	}
```

This is causing the background color to be incorrectly computed and causing false positives e.g. https://github.com/GoogleChrome/lighthouse/issues/1615#issuecomment-285206469

**Comment by alice** [comment]
This may have been missed previously as typically width > height, except when testing in device mode or when devtools is docked to the side.

**Comment by alice** [comment]
Oh right this is just #272

**Comment by marcysutton** [comment]
Merging #272 got stalled due to our lack of mobile tests. @dylanb should we go ahead and merge that PR? There is an open issue for mobile tests here: https://github.com/dequelabs/axe-core/issues/277
